packages/now-build-utils/src/detect-builders.ts

```diff
@@ -33,8 +33,13 @@ function getApiBuilders({ tag }: Pick<Options, 'tag'> = {}): Builder[] {
   ];
 }
 
-function hasDirectory(fileName: string, files: string[]) {
-  return files.some(name => name.startsWith(`${fileName}/`));
+function hasDirectory(name: string, files: string[]) {
+  return files.some(file => file.startsWith(`${name}/`));
+}
+
+function hasBuildScript(pkg: PackageJson | undefined) {
```

Comment:
```suggestion
function hasBuildScript(pkg: PackageJson | null | undefined) {
```

This will fix the build error

Reply:
Ah, already removed it :D